apps/openmw/mwgui/savegamedialog.cpp

```diff
@@ -361,6 +378,11 @@ namespace MWGui
             << MWBase::Environment::get().getWorld()->getMonthName(mCurrentSlot->mProfile.mInGameTime.mMonth)
             <<  " " << hour << " " << (pm ? "#{sSaveMenuHelp05}" : "#{sSaveMenuHelp04}");
 
+        if (Settings::Manager::getBool("timeplayed","Saves"))
+        {
+            text << "\n" << "Time played: " << formatTimeplayed((int)floor(mCurrentSlot->mProfile.mTimePlayed));
```

Comment:
Using ```int``` in one place and ```long int``` in another place is inconsistent. Also, why not just use a ```double```, mTimePlayed is a ```double``` after all. Why the need to cast anything?